### PR TITLE
Only test NODAL FeatureFloodCount in serial

### PR DESCRIPTION
**Pull request by roystgnr**
It's currently failing in parallel, and using the (default) ELEMENTAL
option ought to be an adequate workaround so we aren't likely to fix
NODAL any time soon.

Workaround for #9886

### Complete each of the following items before creating a PR

- Include any relevant design information for your change
- Follow [Coding Standards](http://mooseframework.org/wiki/CodeStandards/)
- Submit one or more [Test Cases](http://mooseframework.org/wiki/MooseTraining/testing/) or modify existing test cases
- Reference or close a specific issue (refs # or closes #)

### CODE REVIEW

File: modules/phase_field/test/tests/boundary_intersecting_features/tests [hunk 1]
```diff
@@ -5,5 +5,6 @@
     csvdiff = boundary_intersecting_features_out_grain_volumes_0000.csv
     # This test requires VTK because it uses the ImageFunction class
     vtk = true
+    max_parallel = 1 # See #9886
```
Comment (permcody) [inline modules/phase_field/test/tests/boundary_intersecting_features/tests]: This test doesn't use the NODAL mode. It does however contain a misleading AuxKernel with the word nodal in it. Did you by chance get a false positive here?

**Review by permcody — COMMENTED** [review]
Oh, and I'm going to conflict with you on my misc reorg PR. No big deal. I'll either fix it up or merge yours first.

File: modules/phase_field/test/tests/boundary_intersecting_features/tests (hunk 1 above)
Comment (roystgnr) [reply modules/phase_field/test/tests/boundary_intersecting_features/tests]: Yeah, I did.  Reenabling now.

Comment (roystgnr) [reply modules/phase_field/test/tests/boundary_intersecting_features/tests]: False positive in the "trying to grep and read through test input files" sense, I mean.

**Comment by permcody** [comment]
Sweet - I totally forced pushed to your fork!

**Comment by permcody** [comment]
I was so excited about forcing pushing to your branch, that I screwed the conflict resolution. Hang on, I'll fix.

**Comment by permcody** [comment]
Doc failure is due to us running out of space on the remote server. All other tests passed - merging.

**Review by permcody — APPROVED** [review]

**Pull request merged**